website/docs/basic-operations/read.md

```diff
@@ -184,21 +169,21 @@ To retrieve all documents containing a specific value in an array, use dot notat
 The following command retrieves all documents containing `"blender"` in the third field of an array:
 
 ```js
-db.employees.find({"catalog.2": "blender"})
+db.employees.find({ 'catalog.2': 'blender' })
 ```
 
 The document that matches the array query is displayed in the response:
 
-```js
+```sh
 [
   {
     _id: ObjectId("636b3b0e0466c61a229bbf9c"),
-    name: { first: 'Sam', last: 'Johnson' },
+    name: { first: "Sam", last: "Johnson" },
     employeeID: 2234,
     age: 35,
-    role: 'salesperson',
-    catalog: [ 'cabinet', 'fridge', 'blender', 'utensils' ]
-  }
+    role: "salesperson",
+    catalog: ["cabinet", "fridge", "blender", "utensils"],
+  },
 ]
 ```
```

Comment:
Please pull and revert changes like this one, but use `json5` instead of `js` or `sh`. The output of commands should not change.